atomicfu/src/jvmMain/kotlin/kotlinx/atomicfu/AtomicFU.kt

```diff
@@ -20,7 +20,7 @@ import java.util.concurrent.atomic.AtomicReferenceFieldUpdater
  * private val f = atomic<Type>(initial)
  * ```
  */
-public actual fun <T> atomic(initial: T): AtomicRef<T> = AtomicRef<T>(initial)
+public actual fun <T> atomic(initial: T, trace: Trace): AtomicRef<T> = AtomicRef<T>(initial)
```

Comment:
The tracing on JVM is not actually implemented yet. It should pass `trace` to `AtomicRef`, which would use to append messages to trace on every change of value. Same for `AtomicInt` and `AtomicLong`.